tests/faucet_mininet_test_unit.py

```diff
@@ -3297,3 +3297,61 @@ def test_switching(self):
         source_host, overridden_host, rewrite_host = self.net.hosts[0:3]
         self.verify_dest_rewrite(
             source_host, overridden_host, rewrite_host, overridden_host)
+
+
+class FaucetDisableHardTimeoutTest(FaucetUntaggedTest):
+    CONFIG_GLOBAL = """
+vlans:
+    100:
+        description: "untagged"
+"""
+    CONFIG = """
+        timeout: 10
+        hard_timeout_enabled: false
+        interfaces:
+            %(port_1)d:
+                native_vlan: 100
+                description: "b1"
+            %(port_2)d:
+                native_vlan: 100
+                description: "b2"
+            %(port_3)d:
+                native_vlan: 100
+                description: "b3"
+            %(port_4)d:
+                native_vlan: 100
+                description: "b4"
+"""
+
+    def test_untagged(self):
+        self.net.pingAll()
+        first_host, second_host = self.net.hosts[:2]
+        third_host, fourth_host = self.net.hosts[2:]
+        first_host.cmd('ping %s &' % second_host.IP())
+        time.sleep(30) # wait enough for hosts to expire
```

Comment:
Is there a way we can monitor the of channel so we can see the expiry messages, rather than just waiting for 30 seconds. That is a very long wait in the test suite..